deploy/submit.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Markus Dolensky, ICRAR, 2020-10-01
+#
+# Usage: sbatch leapjob.sh [OPTIONS]
+#  Options:
+#  -h,--help                   Print this help message and exit
+#  -s,--stations INT           Override number of stations to use in the measurement set
+#  -f,--filepath TEXT          MeasurementSet file path
+#  -d,--directions TEXT        Direction calibrations
+#  -i,--implementation TEXT    Compute implementation type
+#  -c,--config TEXT            Config filepath
```

Comment:
This may be prone to becoming outdated

Reply:
Does not seem that this should be a blockage. More like tech. debt

Reply:
@calgray , had missed the comment before the PR was merged: Yes, I thought about just saying that it passes through the args to leap-accelerate, rather than exposing them explicitly and might do that in the next release.